### PR TITLE
Width height edit date

### PR DESCRIPTION
**Pull request by gdey**
#41 and #42 
@roblabs you may want to look at this as well.

### CODE REVIEW

File: atlante/server/server.go [hunk 1]
```diff
@@ -196,10 +198,19 @@ func encodeCellAsJSON(w io.Writer, cell *grids.Cell, pdf string, lat, lng *float
 	if !lastGen.IsZero() {
 		jsonCell.LastGen = lastGen.Format(time.RFC3339)
 	}
-	pubdate, err := cell.PublicationDate()
-	if err != nil && !pubdate.IsZero() {
-		jsonCell.LastEdited = pubdate.Format(time.RFC3339)
+	/*
```
Comment (ARolek) [inline atlante/server/server.go]: leftover comment?

File: atlante/sheet.go [hunk 2]
```diff
@@ -81,6 +96,28 @@ func (sheet *Sheet) Execute(wr io.Writer, tplContext GridTemplateContext) error
...
+	var mm float64
+	if sheet == nil {
+		// return the default points
+		mm = DefaultHeightMM
+	}
+	mm = sheet.Height
+	return mm * float64(dpi) / mm2inch
+}
+
+// WidthInPoints returns the height in points given the dpi (dots per inch)
```
Comment (ARolek) [inline atlante/sheet.go]: height -> width in comment

Comment (roblabs) [reply atlante/sheet.go]: return mm * float64(dpi) * mm2inch

File: atlante/sheet.go [hunk 3]
```diff
@@ -81,6 +96,28 @@ func (sheet *Sheet) Execute(wr io.Writer, tplContext GridTemplateContext) error
...
+func (sheet *Sheet) WidthInPoints(dpi uint) float64 {
+	var mm float64
+	if sheet == nil {
+		// return the default points
+		mm = DefaultWidthMM
+	}
+	mm = sheet.Width
+	return mm * float64(dpi) / mm2inch
```
Comment (roblabs) [inline atlante/sheet.go]: return mm * float64(dpi) * mm2inch

**Pull request merged**